fe/src/shared/components/MenuTiles/MenuTiles.scss

```diff
@@ -1,10 +1,21 @@
 .menu-tiles {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: center;
+    margin: 0 auto;
```

Comment:
**suggestion:** let's remove these changes from this PR and do it in a separate one. I have a feeling we need to test it a lot.